test/central_shared_services_no_tgw_association_propagation/go.mod

```diff
@@ -0,0 +1,59 @@
+module shared-services-test
```

Comment:
Please remove any go.mod or go.sum files. I know its frustrating but we do not keep *any* go mod or sum files in the code base. They should be included in the `.gitignore` also. The reasoning comes from security alerting in the dependency chain.

I realize that may seem unintuitive but that is our current requirement

Not a requirement for this PR but a FYI, we are going to be transitioning away from `terratest` to `terraform test` after a few more patch versions come out. We've already begun the process in the VPC module: https://github.com/aws-ia/terraform-aws-vpc/pull/127/files

Reply:
.gitignore is updated and go.mod and go.sum files are removed.